.circleci/config.yml

```diff
@@ -16,18 +16,18 @@ commands:
                     sudo apt update
                     sudo apt install python3-requests
                 when: always
-    setup_node16:
-        description: Setup Node 16
+    setup_node18:
```

Comment:
should we rename this step to just `setup_node` to make the upgrade easier next time?

Reply:
yeah, I like that idea